hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -228,6 +228,10 @@ public Operation convert(String value) throws ParameterException {
         + " source-fetch -> Transform -> Hudi Write in loop")
     public Boolean continuousMode = false;
 
+    @Parameter(names = {"--update-payload-class"}, description = "Update payload class in hoodie.properties file if needed, "
```

Comment:
can we do this check automatically by reading the `hoodie.properties` file value and only overwrite if the class name is different?  I am not sure adding an option is the way to go here..

Reply:
@vinothchandar someone might give different payload class by mistake as well. Just to be sure he/she really wants to update it in hoodie.properties file, I added this flag here. The same was discussed with @n3nash.

Reply:
@pratyakshsharma I actually agree with @vinothchandar, we should just check `hoodie.properties` and overwrite if it's different. We don't necessarily want to create guards if someone does actually provide a different class by mistake

Reply:
Done with the changes @n3nash